packages/controller-utils/src/types.ts

```diff
@@ -47,31 +47,30 @@ export enum BuiltInNetworkName {
 }
 
 /**
- * Decimal string chain IDs of built-in networks, by name.
- *
- * `toHex` not invoked to avoid cyclic dependency
+ * Caip-2 chain IDs of built-in networks, by name.
  */
-export const ChainId = {
-  [BuiltInNetworkName.Mainnet]: '0x1', // toHex(1)
-  [BuiltInNetworkName.Goerli]: '0x5', // toHex(5)
-  [BuiltInNetworkName.Sepolia]: '0xaa36a7', // toHex(11155111)
-  [BuiltInNetworkName.Aurora]: '0x4e454152', // toHex(1313161554)
-  [BuiltInNetworkName.LineaGoerli]: '0xe704', // toHex(59140)
-  [BuiltInNetworkName.LineaMainnet]: '0xe708', // toHex(59144)
+export const BuiltInCaipChainId = {
+  [BuiltInNetworkName.Mainnet]: 'eip155:1',
+  [BuiltInNetworkName.Goerli]: 'eip155:5',
+  [BuiltInNetworkName.Sepolia]: 'eip155:11155111',
+  [BuiltInNetworkName.Aurora]: 'eip155:1313161554',
+  [BuiltInNetworkName.LineaGoerli]: 'eip155:59140',
+  [BuiltInNetworkName.LineaMainnet]: 'eip155:59144',
 } as const;
-export type ChainId = typeof ChainId[keyof typeof ChainId];
 
 /**
  * Decimal string network IDs of built-in Infura networks, by name.
  */
-export const NetworkId = {
+export const InfuraNetworkId = {
+  // should these be eip'd?..
```

Comment:
no, right? this is for legacy/compatibility?

Reply:
Correct, shouldnt touch these.